uisdk/src/main/res/values/dimens.xml

```diff
@@ -143,4 +143,12 @@
     <dimen name="kh_uisdk_spacing_grid_row">12dp</dimen>
     <dimen name="kh_uisdk_spacing_grid_second_column">28dp</dimen>
     <dimen name="kh_uisdk_spacing_grid_first_column">16dp</dimen>
+
+    <!--Loyalty -->
+    <dimen name="kh_uisdk_loyaltyViewBalance_paddingTop">3dp</dimen>
```

Comment:
May I suggest to use 
`kh_uisdk_loyalty_balance_padding_top`
[ same for the other ones as well ]